### PR TITLE
Fix typos in tests/

### PR DESCRIPTION
**Pull request by Gelma**

### CODE REVIEW

File: tests/zfs-tests/tests/functional/inuse/inuse_004_pos.ksh [hunk 1]
```diff
@@ -48,7 +48,7 @@ verify_runnable "global"
 function cleanup
 {
 	#
-	# Essentailly this is the default_cleanup routine but I cannot get it
+	# Essentially this is the default_cleanup routine but I cannot get it
 	# to work correctly.  So its reproduced below.  Still need to full
```
[the remark this answers is not in the document]
Comment (Gelma) [reply tests/zfs-tests/tests/functional/inuse/inuse_004_pos.ksh]: Done!

**Comment by freqlabs** [comment]
You will need to squash these to a single commit and remember to sign-off in the commit message.

File: tests/zfs-tests/tests/functional/history/history_005_neg.ksh [hunk 2]
```diff
@@ -44,7 +44,7 @@
 # STRATEGY:
 #	1. Create a test pool.
 #	2. Separately invoke zpool list|status|iostat
-#	3. Verify they was not recored in pool history.
+#	3. Verify they was not recorded in pool history.
```
[the remark this answers is not in the document]
Comment (Gelma) [reply tests/zfs-tests/tests/functional/history/history_005_neg.ksh]: Done and squashed!

File: tests/zfs-tests/tests/functional/history/history_005_neg.ksh [hunk 3]
```diff
@@ -42,9 +42,9 @@
 #		zpool iostat
 #
 # STRATEGY:
-#	1. Create a test pool.
```
Comment (rlaager) [inline tests/zfs-tests/tests/functional/history/history_005_neg.ksh]: Why remove the period here (and in 3)?

Comment (Gelma) [reply tests/zfs-tests/tests/functional/history/history_005_neg.ksh]: Just to have all three the same. Not one with full stop and other not.

**Pull request merged**